flake.nix

```diff
@@ -13,17 +13,13 @@
     # We intentionally import nixpkgs and haskell.nix as non-flakes, to match the
     # flake-free normal build workflow exactly.
     nixpkgs = {
-      type = "github";
-      owner = "NixOS";
-      repo = "nixpkgs";
-      ref = "nixpkgs-unstable";
+      url = "github:NixOS/nixpkgs";
```

Comment:
This seems unlikely to be right. You almost certainly want to follow at least `nixpkgs-unstable`, not master of nixpkgs!!

Reply:
I mirrored exactly what we have in `plutus`, should I change it here/there/both?

Reply:
IMO what's in `plutus` is also wrong then, sorry I missed that. What's there _could_ be reasonable if you're always going to bump nixpkgs to specific known commit, but it's just asking for people to update following nixpkgs master, which is almost always a bad idea (`nixpkgs-unstable` only advances when some checks pass, so is _much_ less likely to be broken than master!).

Reply:
ok I'll change it 👍